lib/Styles/Granite/Sidebar.scss

```diff
@@ -0,0 +1,17 @@
+.sidebar {
+    background-color: bg-color(3);
```

Comment:
Using background color blocks the inset box-shadow on windows. You can see it in dark mode. So we need to add `box-shadow: highlight();` here.

But beware that if you have the whole pane with `.sidebar` style class and the it contains Gtk.StackSidebar then you get double box shadow. So specifically `stacksidebar.sidebar` needs no box shadow and probably no background so it doesn't cover up the shadow again. Alternatively you could clear styles for `.sidebar .sidebar`